src/Symfony/Component/Notifier/Bridge/FakeSms/FakeSmsTransportFactory.php

```diff
@@ -40,6 +41,10 @@ public function create(Dsn $dsn): FakeSmsEmailTransport|FakeSmsLoggerTransport
         $scheme = $dsn->getScheme();
 
         if ('fakesms+email' === $scheme) {
+            if (null === $this->mailer) {
+                throw new MissingFactoryDependencyException($scheme, MailerInterface::class);
```

Comment:
```suggestion
                throw new \LogicException('.....; try running "composer require symfony/mailer".');
```

Thats how we do it Symfony across the codebase, no need for a new Exception here

Reply:
Let me know what you think about the adapted approach.

Reply:
Like it, thanks!